www/src/components/shell/onboarding/SplashToLogoTransition.js

```diff
@@ -129,13 +135,17 @@ function LogoOnly({ children }) {
 
 export function LogoAndSplash({ splashTimeout = 1200, childIsReady = false, children }) {
   const [splashTimerDone, setSplashTimerDone] = useState(false)
-  
+
   useEffect(() => {
-    setTimeout(() => {
+    const timeoutId = setTimeout(() => {
       setSplashTimerDone(true)
     }, splashTimeout)
+
+    return () => {
```

Comment:
can't this be one-lined: `() => clearTimoeut(timeoutId)`

Reply:
yes it can!